src/deploy/functions/release/planner.ts

```diff
@@ -235,6 +235,8 @@ export function checkForIllegalUpdate(want: backend.Endpoint, have: backend.Endp
       return "a scheduled";
     } else if (backend.isTaskQueueTriggered(e)) {
       return "a task queue";
+    } else if (backend.isBlockingTriggered(e)) {
+      return "a blocking";
```

Comment:
We should be probably using event type here. I didn't have event types for eventTriggers because I wasn't sure when/how we'll be able to upgrade from v1 types to v2 types.